data/json/monsters/fish.json

```
@@ -660,6 +696,23 @@
     "upgrades": { "half_life": 14, "into": "mon_fish_pickerel" },
     "name": { "str_sp": "fry" }
   },
+  {
+    "id": "mon_fish_pickerel_american",
+    "type": "MONSTER",
+    "copy-from": "mon_fish_tiny",
+    "name": { "str": "American pickerel" },
+    "description": "An American pickerel.  It looks like a pike, but unlike its chain cousin, this one is way smaller.",
```

```suggestion
    "description": "An American pickerel.  It looks like a pike, but unlike its chain pickerel cousin, this one is way smaller.",
```
Reads better to people not familiar with the fish species